test/Microsoft.Sbom.Api.Tests/Executors/LicenseInformationFetcherTests.cs

```diff
@@ -16,19 +17,20 @@ namespace Microsoft.Sbom.Api.Executors.Tests;
 public class LicenseInformationFetcherTests
 {
     private readonly Mock<ILogger> mockLogger = new Mock<ILogger>();
```

Comment:
I generally advise that people only create Mocks with `MockBehavior.Strict`, because otherwise we might not be actually validating anything about what the mock does. That makes Mock's particularly annoying for things like ILogger, which is why we [created the NoopLogger](https://dev.azure.com/mseng/1ES/_git/SbomStorageService?path=/SbomStorageApi.Test/NoopLogger.cs).